From eabcc8a6e39dc3c28faee3e04aa56cc65a38d65a Mon Sep 17 00:00:00 2001
From: Urja Rannikko <urjaman@gmail.com>
Date: Sun, 19 Jul 2020 20:28:27 +0300
Subject: [PATCH] Add extern to char* outfile declaration to fix build with gcc
 10

Without this linking fails with a rather long spew of
"multiple definition of `outfile'".

Signed-off-by: Urja Rannikko <urjaman@gmail.com>
---
 genisoimage/genisoimage.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/genisoimage/genisoimage.h b/genisoimage/genisoimage.h
index bbedfb0..82c859b 100644
--- a/genisoimage/genisoimage.h
+++ b/genisoimage/genisoimage.h
@@ -376,7 +376,7 @@ extern int	use_fileversion;
 extern int	split_SL_component;
 extern int	split_SL_field;
 extern char	*trans_tbl;
-char		*outfile;
+extern char	*outfile;
 
 #define	JMAX		64	/* maximum Joliet file name length (spec) */
 #define	JLONGMAX	103	/* out of spec Joliet file name length */
-- 
2.27.0

